ci/docker-compose-build.yaml

```diff
@@ -15,16 +15,29 @@
 version: '3.5'
 
 services:
-  tcf:
-    image: tcf-ci-build:${ISOLATION_ID}
+  avalon-shell:
+    image: avalon-shell:${ISOLATION_ID}
     build:
       context: ..
-      dockerfile: docker/Dockerfile.tcf-dev
+      dockerfile: docker/Dockerfile
       args:
         - http_proxy
         - https_proxy
         - no_proxy
```

Comment:
Is avalon-shell is used to run clients ? Then shell container should be created after  starting avalon components like listener and enclave manager. Add a dependency.

Reply:
Yes, its used to run clients and tests. Client depends on the db to be up and running along with enclave manager and listener. But adding enclave manager and listener as dependencies make sense because they are dependent on lmdb service to be up. I will make the change.